Added prefab StartPoint

Kjusok opened the pull request:
Added prefab StartPoint

Kjusok committed:
Added prefab StartPoint

Added prefab StartPoint

Kjusok commented:
I don't understand if I implemented the "Set position" method correctly
Now it's empty!

Kjusok committed:
Added method "SetPossition"

Added method "SetPossition".
When loading a level, the "StartPoint" and "Player" positions are the same.

ega-ega commented on Assets/Prefabs/Gameplay/StartPoint.prefab:
Pls move this file to subfolder "LevelElements"

Kjusok replied:
Done

Kjusok committed:
move prefab "StartPoint" to subfolder "LevelElements"

move prefab "StartPoint" to subfolder "LevelElements"

Kjusok committed:
added OffsetY

added OffsetY
Now the player's position is exactly above the surface of the level
